src/databricks/labs/ucx/queries/assessment/main/30_3_job_details.sql

```diff
@@ -1,12 +1,26 @@
-/* --title 'Job Details' --filter job_name --width 6 */
+/*
+--title 'Job Details'
+--filter job_name
+--width 6
+--overrides '{"spec":{
+    "encodings":{
+      "columns": [
+        {"fieldName": "job_name", "booleanValues": ["false", "true"], "linkUrlTemplate": "/jobs/{{ job_id }}", "linkTextTemplate": "{{ @ }}", "linkTitleTemplate": "{{ @ }}", "linkOpenInNewTab": true, "type": "string", "displayAs": "link", "title": "job_name"},
+        {"fieldName": "job_id", "booleanValues": ["false", "true"], "type": "integer", "displayAs": "number", "title": "job_id"},
```

Comment:
mark job_id column as hidden

Reply:
adding it to invisible columns did not hide it, removed it from the encoding. lmk if removing is not the right approach